src/main/c/logging.c

```diff
@@ -25,7 +25,7 @@
 # define DIR_SEP '/'
 #endif
 
-#define LOGGER_NAME "powerwaf_native"
+#define LOGGER_NAME "Waf_native"
```

Comment:
```suggestion
#define LOGGER_NAME "ddwaf_native"
```

(This is the name that will be visible in logs)

also keep the code convention on the C side where function names are all lowercase (except for JNI signatures).